src/DssSpell.sol

```diff
@@ -52,42 +66,50 @@ contract DssSpellAction is DssAction {
     // A table of rates can be found at
     //    https://ipfs.io/ipfs/QmefQMseb3AiTapiAKKexdKHig8wroKuZbmLtPLv4u2YwW
     //
+    uint256 constant ZERO_PCT =            1000000000000000000000000000;
+    uint256 constant ZERO_POINT_FIVE_PCT = 1000000000158153903837946257;
+    uint256 constant ONE_PCT =             1000000000315522921573372069;
+    uint256 constant ONE_POINT_FIVE_PCT =  1000000000472114805215157978;
+    uint256 constant TWO_PCT =             1000000000627937192491029810;
+    uint256 constant SIX_PCT =             1000000001847694957439350562;
 
     function actions() public override {
-        address MCD_VAT         = DssExecLib.vat();
-        address ILK_REGISTRY    = DssExecLib.getChangelogAddress("ILK_REGISTRY");
-
-        // ---------- Increase the Dust Parameter for ETH-B Vault Type ---------
-
-        DssExecLib.setIlkMinVaultAmount("ETH-B", 30_000);
-
-        // -------------------- Increase the Dust Parameter --------------------
-
-        bytes32[] memory ilks = IlkRegistryLike(ILK_REGISTRY).list();
-        for (uint256 i = 0; i < ilks.length; i++) {
-            bytes32 ilk = ilks[i];
-            if (
-                   ilk == "PSM-USDC-A"
-                || ilk == "ETH-B"
-                || ilk == "ETH-C"
-                || ilk == "RWA001-A"
-                || ilk == "RWA002-A"
-            )  {
-                continue;
-            }
-            DssExecLib.setIlkMinVaultAmount(ilk, 10_000);
-        }
-
-        // --------------------------- Add MCD_FLASH ---------------------------
-
-        Fileable(MCD_FLASH).file("max", 500_000_000 * WAD);
-        Fileable(MCD_FLASH).file("toll", 5 * WAD / 10000);
-        DssExecLib.authorize(MCD_VAT, MCD_FLASH);
-        DssExecLib.setChangelogAddress("MCD_FLASH", MCD_FLASH);
-
-        // -------------------------- Update Chainlog --------------------------
 
-        DssExecLib.setChangelogVersion("1.9.1");
+        // ----------- Auto-Line updates -----------
+        // https://vote.makerdao.com/polling/QmZz4ssm?network=mainnet#poll-detail
```

Comment:
It should be rather: https://vote.makerdao.com/polling/QmfZWY87?network=mainnet#poll-detail?

Reply:
This is the correct link. This is where we update the auto-line parameters for the PSM.

Reply:
Right.